### ISSUE TITLE
Accessing Blago Miner

### ISSUE THREAD

**Issue by Te55er4ct** [PoC-Consortium/Qbundle#25]
When trying to access Blago Miner for the first time it ask's "do you want to download and install it?"

-click yes

Error Popup says
"something went wrong, internet connection might have been lost."

Did not have this issue in previous build version 1.8

**Comment by Quibus-burst** [comment]
You have an antivirus software that blocks the download of the miner or deletes it after it is installed. Please make an exclude for Qbundle and its folder. Then delete the BlagoMiner folder within qbundle if you have it. Restart and try again.

**Comment by Te55er4ct** [comment]
yup, did all of that a dozen times. cant figure it out. So I just copied
the BlagoMiner from another system I had running 1.8.

**Comment by beefycheddar** [comment]
I also have this problem. I didn't have version 1.8, so I can't copy the folder from an older miner. I found Blago on github ([https://github.com/Blagodarenko/miner-burst/releases]). Is this the same one? If so, how do I get qbundle to recognize the extracted/unextracted zip file? I renamed it to the same BlagoMiner_1.170820.zip and folder BlagoMiner_1.170820, but no dice. And, does the Blago folder go in the root of qbundle?
